site/usage/compile.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: usage
+menu: usage
+title: Compiling Vega-Lite
+permalink: /usage/compile.html
+---
+
+If you would rather hand compile your Vega-Lite specifications into Vega, you can use Vega-Lite's provided `vl.compile` function.
+
+First, load the required libraries (D3, Vega, Vega-Lite).
+
+```html
+<script src="//d3js.org/d3.v3.min.js" charset="utf-8"></script>
+<script src="//vega.github.io/vega/vega.js" charset="utf-8"></script>
+<script src="//vega.github.io/vega-lite/vega-lite.js" charset="utf-8"></script>
+```
+
+We suggest that you install Vega-Lite with [npm](https://www.npmjs.com/package/vega-lite) to get the latest stable version. To install Vega-Lite with npm, simply install it as you would any other npm module.
+
+```sh
+npm install vega-lite
+```
+
+Alternatively you can [download the latest Vega-Lite release](https://github.com/vega/vega-lite/releases/latest) and add it to your project manually.
+
+Then, create an HTML element that the visualization should be attached to.
+
+```html
```

Comment:
Can we distinguish the compile and embed pages a bit more distinct? Maybe the embed page should show all steps and the compile page only shows you how to generate vega and then we can link to the vega instructions. I don't think many people want to generate vega (and some might even want to do this on node). 

Let's target a different audience with this page.